aws-distributed-configuration/src/main/java/io/micronaut/aws/distributedconfiguration/AwsDistributedConfiguration.java

```diff
@@ -34,6 +35,13 @@ public interface AwsDistributedConfiguration {
     @NonNull
     String getPrefix();
 
+    /**
+     * @return List of prefixes for AWS Distributed Configuration resources names. If it is non-empty,
+     * {@link AwsDistributedConfiguration#getPrefix()} is not used
+     */
+    @NonNull
+    List<String> getPrefixes();
```

Comment:
Please add `since` tag pointing to `3.12.1`, and make it a `default` method that returns an empty list.